lib/app/modules/home/views/home_view.dart

```diff
@@ -307,19 +309,23 @@ class HomeView extends GetView<HomeController> {
           child: Column(
             children: [
               DrawerHeader(
-                  decoration: const BoxDecoration(color: kLightSecondaryColor),
-                  child: Center(
-                    child: Row(
-                      children: [
-                        const CircleAvatar(
-                            radius: 30,
-                            backgroundImage: AssetImage(
-                              'assets/images/ic_launcher-playstore.png',
-                            )),
-                        const SizedBox(
-                          width: 10,
+                decoration: const BoxDecoration(color: kLightSecondaryColor),
+                child: Center(
+                  child: Row(
+                    children: [
+                      const Flexible(
+                        flex: 1,
+                        child: CircleAvatar(
+                          radius: 30,
+                          backgroundImage: AssetImage(
+                            'assets/images/ic_launcher-playstore.png',
+                          ),
                         ),
-                        Column(
+                      ),
+                      const Gap(10),
```

Comment:
I don't think it's a good idea to rely on a new package only to use it once over here. We can go ahead with either padding or sizedbox itself in this case.

Reply:
For sure I will change it to a sizedbox and update the pr

Reply:
I have made the changes and update the PR